src/lite-yt-embed.js

```diff
@@ -99,10 +99,12 @@ class LiteYTEmbed extends HTMLElement {
     }
 
     addIframe(){
+        const params = new URLSearchParams(this.getAttribute('params'));
```

Comment:
```suggestion
        const params = new URLSearchParams(this.getAttribute('params') || []);
```

if nothing was defined, we'd be using the `null` return value.. 

![image](https://user-images.githubusercontent.com/39191/96809157-2b448180-13cf-11eb-8785-57ca040a5797.png)